gsitemap.php

```diff
@@ -48,7 +48,7 @@ public function __construct()
         $this->description = $this->trans('Generate your Google sitemap file', array(), 'Modules.Gsitemap.Admin');
         $this->confirmUninstall = $this->trans('Are you sure you want to uninstall this module?', array(), 'Admin.Notifications.Warning');
         $this->ps_versions_compliancy = array('min' => '1.7', 'max' => _PS_VERSION_);
-        $this->type_array = array('home', 'meta', 'product', 'category', 'manufacturer', 'supplier', 'cms', 'module');
+        $this->type_array = array('home', 'meta', 'product', 'category', 'cms', 'module');
```

Comment:
Please don't delete these. The Manufacturer and Supplier pages are toggled by the setting in "Shop Parameters" > "General" > "Display brands and suppliers"

Reply:
Fixed